spec.bs

```diff
@@ -7728,8 +7769,9 @@ An <dfn export>auction config</dfn> is a [=struct=] with the following [=struct/
     uniquely associated with all {{AuctionAdConfig/componentAuctions}}.
     This must come from a prior call to {{Window/navigator}}.{{Navigator/createAuctionNonce()}}.
     This is only required for auctions that provide [=additional bids=], and each of those
-    [=additional bids=] must use the same auction nonce to ensure that each of them was intended for
-    this and only this auction.
+    [=additional bids=] must use the same auction nonce (or the bid nonce derived from that auction

```

Comment:
Similarly, can we make bid nonce the primary behavior, e.g.

```
[=additional bids=] must include the same bid nonce derived from that auction nonce as computed
by [=calculate expected bid nonce=]) to ensure that each [=additional bid=] was intended for this
and only this auction. For backwards compatibility, an [=additional bid=] may include the
[=auction nonce=] directly in place of a bid nonce.
```

Reply:
Done.